cmd/tke-installer/app/installer/installer.go

```diff
@@ -1001,6 +1039,11 @@ func (t *TKE) findClusterProgress(request *restful.Request, response *restful.Re
 func (t *TKE) do() {
 	ctx := t.log.WithContext(context.Background())
 
+	// Expansion
+	t.mergeExpansionsProviderConf()
+	t.mergeExpansionsConfig()
+	t.mergeExpansionsCluster()
+
```

Comment:
If we need merge config, we should merge once and forever, no need add new method in future.